packages/valory/skills/abstract_round_abci/behaviours.py

```diff
@@ -236,6 +243,7 @@ def act(self) -> None:
         if self.current_behaviour is None:
             return
 
+        self.__termination_behaviour.act_wrapper()
```

Comment:
Does it make sense here or does it make more sense after 251. The latter would give the `current_behaviour` the option to finish.